package.json

```diff
@@ -15,7 +15,8 @@
     "migrate": "./migrate.js",
     "docker:build": "docker build . -t daostack/migration:$(cat package.json | jq -r '.version')",
     "docker:push": "docker push daostack/migration:$(cat package.json | jq -r '.version')",
-    "release": "./release.sh"
+    "release": "./release.sh",
+    "prune-arc-build": "node ./pruneArcBuild.js"
```

Comment:
Maybe better to add it to `postinstall` script or similar, so it happens automatically whenever you `npm install` the repo.

Reply:
Fixed